plugins/formal-verification/formver.viper/src/org/jetbrains/kotlin/formver/viper/MangledName.kt

```diff
@@ -13,4 +13,6 @@ package org.jetbrains.kotlin.formver.viper
  */
 interface MangledName {
     val mangled: String
+    val isCollection: Boolean
+        get() = false
```

Comment:
Hm, it's a bit annoying to have this all the way here... Does putting it in `ScopedKotlinName` not suffice?  Perhaps we can change some `MangledName`s to `ScopedKotlinName`s?